test/simplify_algebra_test.cpp

```diff
@@ -1782,13 +1823,19 @@ TEST_CASE(simplify_mul_slice_conv_horiz_fusion)
     }
     EXPECT(m1.sort() == m2.sort());
 }
-TEST_CASE(reorder_reshape_slice)
+
+template <std::size_t BS, bool InTrans>
```

Comment:
Can we rename `InTrans` to `TransposeInput`?

Reply:
Done